test/runtests.jl

```diff
@@ -2,7 +2,7 @@ using BraketAHS: run, save_results
 using Test
 using JSON3
 
-@testset "BraketAHS.jl" begin
+@testset "BraketAHS.jl: Non-empty local detunings" begin
     # Write your tests here.
```

Comment:
Maybe we can get rid of this comment lol

Reply:
Removed comment